massa-event-cache/src/controller.rs

```diff
@@ -0,0 +1,43 @@
+use crate::config::EventCacheConfig;
+use crate::event_cache::EventCache;
+use massa_models::execution::EventFilter;
+use massa_models::output_event::SCOutputEvent;
+
+#[allow(dead_code)]
+/// Final event cache controller
+pub struct EventCacheController {
+    /// Cache config.
+    /// See `EventCacheConfig` documentation for more information.
+    cfg: EventCacheConfig,
+    /// Event stored cache.
+    /// See the `EventCache` documentation for more information.
+    event_cache: EventCache,
+}
+
+impl EventCacheController {
+    /// Creates a new `EventCacheController`
+    pub fn new(cfg: EventCacheConfig) -> Self {
+        let event_cache = EventCache::new(
+            cfg.event_cache_path.clone(),
```

Comment:
I think the clone() can be removed
```suggestion
            &cfg.event_cache_path,
```
and in massa-event-cache/src/event_cache.rs:44:
```rust
    pub fn new(
        path: &Path,
```

Reply:
Done